pipelines.py

```diff
@@ -5,8 +5,7 @@
 from steps.preprocessing.misc import XYSplit
 from utils import squeeze_inputs
 from models import PyTorchUNet, PyTorchUNetStream
-from postprocessing import Resizer, CategoryMapper, MulticlassLabeler, MaskDilator, DenseCRF, ScoreBuilder, \
-    ResizerStream, CategoryMapperStream, MulticlassLabelerStream, MaskDilatorStream, DenseCRFStream
+from postprocessing import *
```

Comment:
@taraspiotr I am not a fan of * import. I think it is advised not to use them but I could be wrong. In the situation like this I would just go with import postprocessing as post or something like that